### DISCUSSION TITLE
r/china-travel-layoff-trip-day3-huangshan-yixian

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#82] [Announcements]
# r/china-travel-layoff-trip-day3-huangshan-yixian

皖南黟县究竟是什么样子的？有没有什么好玩儿的？

https://blog.ops-coffee.cn/r/china-travel-layoff-trip-day3-huangshan-yixian

**Comment by wintersun** [comment]
先选择做正确的事，再精进正确地做事
长期主义，无需焦虑！

**Comment by qinze118** [comment]
同为中年运维人，也很想体验这种生活
